app/components/ecosystem/module-applications.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
+import { require as local } from "app-root-path";
```

Comment:
Were you not able to do `import markdown from "../markdown";`? That way of importing should work now.

Reply:
I just copy and pasted what was done elsewhere.